src/Illuminate/Notifications/Channels/SlackWebhookChannel.php

```diff
@@ -111,4 +114,21 @@ protected function fields(SlackAttachment $attachment)
             return ['title' => $key, 'value' => $value, 'short' => true];
         })->values()->all();
     }
+
+    /**
+     * Format the attachment's actions.
+     *
+     * @param  \Illuminate\Notifications\Messages\SlackAttachment  $attachment
+     * @return array
+     */
+    protected function actions(SlackAttachment $attachment)
+    {
+        return collect($attachment->actions)->map(function ($value) {
```

Comment:
Do we really need a `Collection`?

This can be achieved with:
```php
return array_map(function ($action) {
    return $action instanceof SlackAttachmentAction ? $action->toArray() : $action;
}, (array) $attachment->actions);
```